plyer/facades/processors.py

```diff
@@ -1,10 +1,12 @@
 '''
 Number of Processors
 =======
+The :class:`Processors` provides a information on the number of
+processors in a system
```

Comment:
Maybe we should also add a deprecation notice? (We lost our chance to remove `processor` before release `2.0.0`)